server/services/prismic.js

```diff
@@ -224,10 +224,12 @@ function createEventPromos(allResults): Array<EventPromo> {
         id: event.id,
         title: asText(event.data.title),
         url: `/events/${event.id}`,
+        format: event.data.format && parseEventFormat(event.data.format),
         start: eventAtTime.startDateTime,
         end: eventAtTime.endDateTime,
         image: prismicImage(promoImage),
-        description: asText(promoCaption)
+        description: asText(promoCaption),
```

Comment:
I don't believe the description features at all in the promo designs – would have to check with @Norvard  or @Heesoomoon if this should always be the case.